### DISCUSSION TITLE
Error on the Listener (Even though it is exactly the same as the documentation) - Orchid v14

### DISCUSSION THREAD

**Discussion by nuha087** [orchidsoftware/platform#2689] [Q&A]
I just upgraded from version 13 to 14. After successfully upgrading I tried the Listener exactly as in the documentation but got an error of 500. Anyone know the cause?

<img width="416" alt="image" src="https://github.com/orchidsoftware/platform/assets/95839462/404d5db2-ced2-46ee-92d1-d2d5e4fc542d">

**Comment by BNohut** [comment]
Can you share a screenshot which shows _preview_ tab of console?
